opacclient/libopac/src/test/java/de/geeksfactory/opacclient/apis/WiseTest.kt

```diff
@@ -0,0 +1,61 @@
+package de.geeksfactory.opacclient.apis
+
+import de.geeksfactory.opacclient.networking.HttpClientFactory
+import de.geeksfactory.opacclient.objects.Account
+import de.geeksfactory.opacclient.objects.DetailedItem
+import de.geeksfactory.opacclient.objects.Library
+import de.geeksfactory.opacclient.searchfields.SearchQuery
+import de.geeksfactory.opacclient.searchfields.TextSearchField
+import org.json.JSONArray
+import org.json.JSONObject
+import org.junit.Assert.assertTrue
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+import org.mockito.Mockito
+import java.util.*
+
+@RunWith(Parameterized::class)
+class WiseTest(private val file: String) : BaseHtmlTest() {
+    val wise =  Mockito.spy(Wise::class.java)
+
+    init {
+        wise.init(Library().apply {
+            data = JSONObject().apply {
+                put("baseurl", "https://catalogus.biblionetdrenthe.nl")
+                put("imgurl", "/cgi-bin/momredir.pl?")
+                put("api-key-id", "4251d316-b964-44cc-b5fc-2f888d3716a8")
+                put("api-key", "218f81af-dcab-4cca-97f3-a4cffe19a0c9")
```

Comment:
Our general tests should not do real network calls to avoid breaking tests when library servers are down. I see that you're mocking things, which is good, but in that case we can probably also skip hardcoding API keys?